src/tbp/monty/simulators/simulator.py

```diff
@@ -49,17 +49,17 @@ def add_object(
         available objects are preloaded and keyed by name.
 
         Args:
-            name (str): Registered object name
-            position (VectorXYZ): Initial absolute position of the object
-            rotation (QuaternionWXYZ): Initial orientation of the object
-            scale (VectorXYZ): Initial object scale
-            semantic_id (Optional[str]): Optional override object semantic ID
-            enable_physics (bool): Whether to enable physics on the object
-            object_to_avoid (bool): If True, ensure the object is not colliding with
-              other objects
-            primary_target_bb (List | None): If not None, this is a list of the min and
+            name: Registered object name.
+            position: Initial absolute position of the object.
+            rotation: Initial orientation of the object.
+            scale: Initial object scale.
+            semantic_id: Optional override object semantic ID.
```

Comment:
I know I just wrote this, and I might have just copied it from `HabitatSim`, but this just reads poorly. I think it should be something like this instead:
```suggestion
            semantic_id: Optional override for the object's semantic ID.
```